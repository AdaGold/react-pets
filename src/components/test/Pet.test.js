import React from 'react';
import Pet from '../Pet';
import { shallow } from 'enzyme';

describe('Pet', () => {
  test('that it matches an existing snapshot', () => {
    // First Mount the Component in the testing DOM
    // Arrange
    const wrapper = shallow(
      <Pet
        name="test name"
        breed="test breed"
        />
    );

    // Assert that it looks like the last snapshot
    expect(wrapper).toMatchSnapshot();
  });
});
