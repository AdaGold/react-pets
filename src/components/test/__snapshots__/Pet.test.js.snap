// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pet that it matches an existing snapshot 1`] = `
<tr
  className="pet"
>
  <td>
    test name
  </td>
  <td>
    test breed
  </td>
  <td />
  <td />
</tr>
`;
